packages/runtime/test/integration/deleteApp.ts

```diff
@@ -48,20 +48,28 @@ describe("Algorand Smart Contracts - Delete Application", function () {
   });
 
   it("should delete application", function () {
+    const initialMinBalance = john.minBalance;
     const appId = runtime.addApp(flags, {}, approvalProgram, clearProgram);
-    deleteParams.appId = appId;
-    runtime.executeTx(deleteParams);
+    assert.isAbove(runtime.getAccount(john.address).minBalance, initialMinBalance);
+
+    runtime.executeTx({ ...deleteParams, appId: appId });
 
     // verify app is deleted
     expectRuntimeError(
       () => runtime.getApp(appId),
       RUNTIME_ERRORS.GENERAL.APP_NOT_FOUND
     );
+    // minbalance should reduce to initial value after app is deleted
+    assert.equal(john.minBalance, initialMinBalance);
   });
 
   it("should not delete application if logic is rejected", function () {
-    // create app
-    const appId = runtime.addApp(flags, {}, approvalProgram, clearProgram);
+    const initialMinBalance = john.minBalance;
+    const appId = runtime.addApp(flags, {}, approvalProgram, clearProgram); // create app
+
+    const minBalanceAfterAddApp = runtime.getAccount(john.address).minBalance;
+    assert.isAbove(minBalanceAfterAddApp, initialMinBalance); // min balance should increase
```

Comment:
please add how much balance is increased, it is different for opt-in and creation. right ?

Reply:
no, same `0.1ALGO`

Reply:
This one of for asa, I think for app it differs

Reply:
oh yes, thought it was asa. 
For app, cost for creation & optin are different as in creation we increase balance using `global ints/bytes`, while in opt-in we increase/decrease balance referring to `local ints/bytes`. Base fee is same.